sdk/translation/azure-ai-translation-text/samples/Sample6_DictionaryExamples.md

```diff
@@ -17,7 +17,7 @@ try:
     target_language = "es"
     input_text_elements = [ DictionaryExampleTextItem(text = "fly", translation = "volar") ]
 
-    response = text_translator.dictionary_examples(content = input_text_elements, from_parameter = source_language, to = target_language)
+    response = text_translator.lookup_dictionary_examples(content = input_text_elements, from_parameter = source_language, to = target_language)
```

Comment:
Just leaving this here for future reference, the live test pipeline should also be configured to run the samples. This will help check if there are any issues with the samples beforehand.

Reply:
ACK, will make a note.